### DISCUSSION TITLE
New in RaceCar

### DISCUSSION THREAD

**Discussion by jellythomas** [zendesk/racecar#201] [Q&A] [answered]
Hi guys, i'm really new in implementing kafka, i need the kafka as consumer, and i look at this gem, i just want to know.

1. if we want to run the consumer we need to run :
bundle exec racecar FirstConsumer
bundle exec racecar SecondConsumer
bundle exec racecar ThirdConsumer
cant we just run it with one command to run all the consumer or wrap it somehow?

thankyou

**Comment by dasch** [comment, marked as answer]
You need to run three different processes – and that's by design. You want these to operate independently.
